stonesoup/plotter.py

```diff
@@ -369,7 +369,7 @@ def plot_tracks(self, tracks, mapping, uncertainty=False, particle=False, label=
            may be removed in the future.
         """
         label = kwargs.pop('track_label', None) or label
-        tracks_kwargs = dict(linestyle='-', marker="s", color=None)
+        tracks_kwargs = dict(linestyle='-', marker="s", color=None, alpha=0.2)
```

Comment:
This also has the consequence of making the track itself transparent and hard to see. Maybe a separate argument could be added for ellipse alpha value?

Reply:
<img width="565" alt="image" src="https://github.com/user-attachments/assets/153a7c09-05d0-4a44-866d-920670c63f3a" />

Reply:
I see... this is due to
https://github.com/iwishiwasaneagle/Stone-Soup/blob/4b1fb02dea83a1b8fc98add971cafad3a1f73513/stonesoup/plotter.py#L395-L398
Would changing it to `uncertainty_kwargs` be more apt?

Reply:
With `uncertainty_kwargs` as at least that gives most flexibility for anything additional anyone might want. But that has complications with dealing with colour to keep sure that matches the track (unless specified). Whereas `uncertainty_alpha` is simpler and there aren't that many extra arguments other than alpha that seem that useful looking at matplotlib docs.

I think either approach would be fine.

Reply:
`uncertainty_alpha` is now a function kwarg. Running the bearings only example shows that the above bug has been fixed. I was further able to easily set the `uncertainty_alpha` by editing the example.
![image](https://github.com/user-attachments/assets/3bf50da6-3a0e-4765-b2dd-83883b6d2d02)